chart/queue-worker/README.md

```diff
@@ -29,27 +29,40 @@ helm upgrade slow-queue chart/queue-worker \
 ```
 
 ## Configuration
+
 | Parameter | Description | Default |
 |-----------|-------------|---------|
-| `image` | The jetstream-queue-worker image that should be deployed | See values.yaml |
+| `image` | The queue-worker image that should be deployed | See values.yaml |
 | `replicas` | Number of queue-worker replicas to create | `1` |
+| `queueName` | Name of the queue | `faas-request` |
+| `mode` | Queue operation mode: `static` or `function` | `static` |
 | `maxInflight` | Control the concurrent invocations | `1` |
-| `maxWaiting` | Configure the max waiting pulls for the queue-worker JetStream consumer. The value should be at least max_inflight * queue_worker.replicas. Note that this value can not be updated once the consumer is created. | `512` |
+| `queuePartitions` | Number of queue partitions | `1` |
+| `partition` | Queue partition number this queue should subscribe to | `0` |
+| `consumer.inactiveThreshold` | If a function is inactive (has no invocations) for longer than this threshold its consumer will be removed to save resources | `30s` |
+| `consumer.pullMaxMessages` | PullMaxMessages limits the number of messages to be buffered per consumer. Leave empty to use optimized default for the selected queue mode | `` |
 | `upstreamTimeout` | Maximum duration of upstream function call | `1m` |
-| `maxRetryAttempts` | The amount of times to try sending a message to a function before discarding it |`10` |
+| `maxRetryAttempts` | The amount of times to try sending a message to a function before discarding it | `10` |
 | `maxRetryWait` | The maximum amount of time to wait between retries | `120s` |
 | `initialRetryWait` | The amount of time to wait for the first retry | `10s` |
 | `httpRetryCodes` | A comma-separated list of HTTP status codes which the queue worker will retry when received from a function | `408,429,500,502,503,504` |
 | `backoff` | The backoff algorithm used for retries. Must be one off `exponential`, `full` or `equal`| `exponential` |
 | `gateway.host` | The host at which the OpenFaaS gateway can be reached | `http://gateway.openfaas` |
-| `insecureTLS` | Enable insecure tls for callbacks | `false` |
 | `gateway.port` | The port at which the OpenFaaS gateway can be reached | `8080` |
-| `nats.host` | The host at which the NATS JetStream serber can be reached | `nats.openfaas` |
-| `nats.port` | The part at which the NATS JetStream server can be reached | `4222` |
-| `nats.reconnect.attempt` | Max NATS reconnection attempts | `120` |
+| `insecureTLS` | Enable insecure tls for callbacks | `false` |
+| `nats.host` | The host at which the NATS JetStream server can be reached | `nats.openfaas` |
+| `nats.port` | The port at which the NATS JetStream server can be reached | `4222` |
+| `nats.reconnect.attempts` | Max NATS reconnection attempts | `120` |
```

Comment:
Do we still have this?

Reply:
It has been removed from the OpenFaaS chart where I originally added it as part of this PR.
It was already a part of dedicated queue-worker chart. Should it be removed here as well?